src/database/models/travelRequests.js

```diff
@@ -0,0 +1,23 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+export default (sequelize, DataTypes) => {
+  const travelRequests = sequelize.define('travelRequests', {
+    requestId: { type: DataTypes.UUID, allowNull: false },
+    userEmail: { type: DataTypes.STRING, allowNull: false },
+    type: DataTypes.ENUM('one way', 'return'),
```

Comment:
@blaise82 
```suggestion
    type: DataTypes.ENUM('one-way', 'return'),
```
I think it would be easier to make  comparison when you want like to know one-way requests.

Reply:
@hakoemmy please refer the link https://sequelize.org/master/class/lib/data-types.js~ENUM.html#instance-constructor-constructor
sequelize.org giving an example of how ENUMs can be named

Reply:
@blaise82 thanks for your response.